core/src/main/scala/precog/Changes.scala

```diff
@@ -0,0 +1,5 @@
+package precog
+
+class Changes {
+
+}
```

Comment:
Stray file that was never compiled and thus doesn't have headers.

Reply:
It wasn't supposed to be added. It sneaked in when I made the GITHUB_ACTOR change.